components/list/list-item-generic-layout.js

```diff
@@ -154,7 +154,7 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 			}
 
 			::slotted([slot="content-action"]) {
-				grid-column: content-start / end;
+				grid-column: content-start / content-end;
```

Comment:
Comment on this here: https://github.com/BrightspaceUI/core/pull/5465/files/f7e08391628b092e22219aa43d4254f166c595ed#r2012308064